remediation/fence-agents-remediation/configuration.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: default
+title: Configuration
+parent: Fence Agents Remediation
+grand_parent: Remediation
+nav_order: 2
+---
+
+## Fence Agents Remediation Configuration
+
+Fence Agents Remediation (FAR) uses a Custom Resource (CR) created by [Node Health Check Operator](https://github.com/medik8s/node-healthcheck-operator#readme) (NHC) based on a specific remediation template (*FenceAgentsRemediationTemplate*).
+
+Since NHC comes with [Self Node Remediation](https://www.medik8s.io/remediation/self-node-remediation/self-node-remediation/) as the default remediation mechanism, it needs to be configured to use FAR. Thus a user must do the following steps:
+
+* Install FAR and NHC
+* Create FAR Template CR
+* Create NHC CR to use FAR Template CR accordingly (see [FAR documentation](https://github.com/medik8s/fence-agents-remediation#far-with-nhc)).
```

Comment:
Another option is to use NHC's escalation remediation

Reply:
>Create NHC CR to use FAR Template CR accordingly

But writing the above includes NHC's escalation remediation as a user can create NHC CR to solely use FAR template or FAR template with different remediation templates.. 
Do you still see a point of pointing this out here? I might add this in https://github.com/medik8s/fence-agents-remediation#far-with-nhc if it isn't included (after the new FAR readme PR) or in NHC documentation.

Reply:
both classic as well as escalating remediation use the template, so IMHO this is fine